pkg/controller/migplan/gvk.go

```diff
@@ -0,0 +1,62 @@
+package migplan
+
+import (
+	migapi "github.com/fusor/mig-controller/pkg/apis/migration/v1alpha1"
+	gvk "github.com/fusor/mig-controller/pkg/gvkcompare"
+)
+
+func (r ReconcileMigPlan) compareGVK(plan *migapi.MigPlan) error {
+	// No spec chage this time
+	if plan.HasReconciled() {
+		return nil
+	}
+
+	plan.Status.InitUnsupported()
+
+	gvkCompare, err := r.prepareGVKCompare(plan)
+	if err != nil {
+		log.Error(err, "Failed to prepare GVK compare")
+		return err
+	}
+
+	err = gvkCompare.Compare()
+	if err != nil {
+		log.Error(err, "Failed to compare GVRs on both clusters")
+		return err
+	}
+
+	if len(plan.Status.UnsupportedNamespaces) > 0 {
+		plan.Status.SetCondition(migapi.Condition{
+			Type:     NotSupported,
+			Status:   True,
+			Category: Warn,
+			Message:  NsNotSupported,
+			Items:    plan.Status.SelectUnsupported(),
+		})
+	}
+
+	return nil
+}
+
+func (r ReconcileMigPlan) prepareGVKCompare(plan *migapi.MigPlan) (*gvk.GVKCompare, error) {
+	gvkCompare := &gvk.GVKCompare{
+		Plan: plan,
+	}
+
+	err := gvkCompare.PrepareSourceDiscovery(r)
+	if err != nil {
```

Comment:
Please add:
```
log.Trace(err)
```

Reply:
Should pick it up: https://github.com/fusor/mig-controller/pull/413/files#diff-ff4cf8d4f8669d3270d95d4c88c43266R17